sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample9_RecognizeCustomEntitiesAsync.cs

```diff
@@ -102,7 +102,6 @@ await foreach (AnalyzeActionsResult documentsInPage in operation.Value)
                             Console.WriteLine($"  Offset: {entity.Offset}");
                             Console.WriteLine($"  Length: {entity.Length}");
                             Console.WriteLine($"  ConfidenceScore: {entity.ConfidenceScore}");
-                            Console.WriteLine($"  SubCategory: {entity.SubCategory}");
```

Comment:
Why are removing this piece of code? Does it make the sample fail somehow?

Reply:
it does not make the sample fail. but its an extra line that's not correct, and we should be removing it in the next update.